lib/Solver/CoreSolver.cpp

```diff
@@ -88,11 +88,15 @@ Solver *createCoreSolver(CoreSolverType cst) {
     return createDummySolver();
   case Z3_SOLVER:
 #ifdef ENABLE_Z3
+    llvm::errs() << "Using Z3 solver backend\n";
     return new Z3Solver();
 #else
     llvm::errs() << "Not compiled with Z3 support\n";
     return NULL;
 #endif
+  case NO_SOLVER:
+    llvm::errs() << "Invalid solver\n";
```

Comment:
I don't want to use ``klee_error`` here becaues that will cause the program to exit. Instead the caller should handle the ``nullptr`` that is returned.